man/wavpack.1

```diff
@@ -103,7 +103,8 @@ On by default in lossless mode and with the
 .Fl cc
 option.
 .It Fl d
-Delete source file if successful.
+delete source file if successful;
```

Comment:
In general, I preffer if the description is valid sentences,
including the capitalization; but these are nits regarding content
(as opposed to markup), which we can address later.